include/Core/Metas/Modules/DeleteTermMetaModule.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace BulkWP\BulkDelete\Core\Metas\Modules;
+
+use BulkWP\BulkDelete\Core\Metas\MetasModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Delete Term Meta.
+ *
+ * @since 6.1.0
+ */
+class DeleteTermMetaModule extends MetasModule {
+	/**
+	 * Initialize the Module.
+	 */
+	protected function initialize() {
+		$this->field_slug    = 'meta_term';
+		$this->meta_box_slug = 'bd-meta-term';
+		$this->action        = 'delete_meta_term';
+		$this->cron_hook     = 'do-bulk-delete-term-meta';
+		$this->messages      = array(
+			'box_label'  => __( 'Bulk Delete Term Meta', 'bulk-delete' ),
+			'scheduled'  => __( 'Term meta fields from the posts with the selected criteria are scheduled for deletion.', 'bulk-delete' ),
+			'cron_label' => __( 'Delete Term Meta', 'bulk-delete' ),
+		);
+	}
+
+	/**
+	 * Render the Modules.
+	 *
+	 * @return void
+	 */
+	public function render() {
+		?>
+		<!-- Term Meta box start-->
+		<fieldset class="options">
+		<?php
+		$taxonomies = $this->get_taxonomies();
+		?>
+		<h4><?php _e( 'Select the taxonomy whose term meta fields you want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+		<?php
+		foreach ( $taxonomies as $taxonomy ) {
+			?>
+			<tr>
+				<td>
+					<input name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy" value = "<?php echo $taxonomy; ?>" type = "radio" class = "smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy">
+					<label for="smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy"><?php echo $taxonomy; ?> </label>
+				</td>
+			</tr>
+			<?php
+		}
+		?>
+		</table>
+
+		<h4><?php _e( 'Choose your term want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+			<tr>
+				<td>
+					<select class="select2 select2-terms" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term">
+						<option>Choose Terms</option>
+					</select>
+				</td>
+			</tr>
+		</table>
+
+		<h4><?php _e( 'Choose your term meta want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+			<tr>
+				<td>
+					<select class="select2 select2-term-meta" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta">
+						<option>Choose Term Meta</option>
+					</select>
+
+					<select name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta_option">
+						<option value="equal">Equal to</option>
+						<option value="not_equal">Not equal to</option>
+					</select>
+
+					<input type="text" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta_value" />
+				</td>
+			</tr>
+		</table>
+
+		<?php
+		/**
+		 * Add more fields to the delete post meta field form.
+		 * This hook can be used to add more fields to the delete post meta field form.
+		 *
+		 * @since 5.4
```

Comment:
Looks like you copy pasted and didn't change.

Replace post meta with term meta and change 5.4 to 6.0.0

Reply:
Fixed